client/src/components/LobbySettings.svelte

```diff
@@ -0,0 +1,141 @@
+<script lang="ts">
+  import "@material/typography/mdc-typography.scss";
+  import Dialog, { Header, Title, Content, Actions } from "@smui/dialog";
+  import Button, { Label } from "@smui/button";
+  import IconButton from "@smui/icon-button";
+  import Slider from "@smui/slider";
+  import FormField from "@smui/form-field";
+  import { GAME_STATE_DURATIONS_MIN, GAME_STATE_DURATIONS_MAX, type Lobby } from "$lib/firebase/firestore-types/lobby";
+  import { applyLobbySettings } from "$lib/firebase/firebase-functions";
+
+  export let lobby: Lobby;
+  export let lobbyCode: string;
+  let errorMessage: string = "";
+  let showLobbySettings = false;
+  let catfishValue: number = 1;
+  let promptTimerValue = lobby.lobbySettings.promptTime;
+  let chatTimerValue = lobby.lobbySettings.chatTime;
+  let voteTimerValue = lobby.lobbySettings.voteTime;
+
+  const sliderLabels = new Map([
+    ["PROMPT", "Prompt Timer"],
+    ["CHAT", "Chat Timer"],
+    ["VOTE", "Vote Timer"],
+  ]);
+  const minimumTimes = new Map([
+    ["PROMPT", GAME_STATE_DURATIONS_MIN.PROMPT],
+    ["CHAT", GAME_STATE_DURATIONS_MIN.CHAT],
+    ["VOTE", GAME_STATE_DURATIONS_MIN.VOTE],
+  ]);
+  const maximumTimes = new Map([
+    ["PROMPT", GAME_STATE_DURATIONS_MAX.PROMPT],
+    ["CHAT", GAME_STATE_DURATIONS_MAX.CHAT],
+    ["VOTE", GAME_STATE_DURATIONS_MAX.VOTE],
+  ]);
```

Comment:
these are unessisary because you could just use `GAME_STATE_DURATIONS_MIN` and `GAME_STATE_DURATIONS_MAX` directly